### ISSUE TITLE
about two branch

### ISSUE THREAD

**Issue by frensher** [ShiqiYu/OpenGait#78]
你好,我想问下,能不能两个分支分别计算loss呢
![图片](https://user-images.githubusercontent.com/42425485/185533899-1d1732a8-2caf-4d19-88ac-b77bc29eb78c.png)
像这个图这样
![图片](https://user-images.githubusercontent.com/42425485/185533970-c90799f0-da11-4c1b-a429-68d5b8160f8a.png)
相当于这块又增加一个字典
这需要怎么实现呢 
是不是需要修改basemodel
谢谢

**Comment by darkliang** [comment]
Yes, you can. You need to pass the feature maps and labels from the two branches into `traning_feat`, like https://github.com/ShiqiYu/OpenGait/blob/3b758021847dd8b277e275ed0a5ffe62ab824a11/opengait/modeling/models/baseline.py#L37-L48

Also you need to modify the configuration file to support two losses, like https://github.com/ShiqiYu/OpenGait/blob/3b758021847dd8b277e275ed0a5ffe62ab824a11/configs/baseline/baseline.yaml#L25-L34

**Comment by frensher** [comment]
> Yes, you can. You need to pass the feature maps and labels from the two branches into `traning_feat`, like
> 
> https://github.com/ShiqiYu/OpenGait/blob/3b758021847dd8b277e275ed0a5ffe62ab824a11/opengait/modeling/models/baseline.py#L37-L48
> 
> Also you need to modify the configuration file to support two losses, like
> 
> https://github.com/ShiqiYu/OpenGait/blob/3b758021847dd8b277e275ed0a5ffe62ab824a11/configs/baseline/baseline.yaml#L25-L34

![图片](https://user-images.githubusercontent.com/42425485/185544985-8683900b-e19a-4d1d-af17-9ce96d5a8008.png)
![图片](https://user-images.githubusercontent.com/42425485/185545010-27001d89-80b1-43ec-b0ea-34d58e2271a3.png)

这样不行啊  是要特征拼接吗

**Comment by darkliang** [comment]
The `key` should all be `embeddings` and `logits` for `triplet` and `softmax`.

**Comment by frensher** [comment]
> The `key` should all be `embeddings` and `logits` for `triplet` and `softmax`.

![图片](https://user-images.githubusercontent.com/42425485/185548106-cd20af43-7b81-440f-b292-4149800bfeef.png)
![图片](https://user-images.githubusercontent.com/42425485/185548125-9de06d11-2499-4448-89e6-63e0760de796.png)
感谢,已经跑通
